nilearn/surface/surface.py

```diff
@@ -1119,8 +1119,8 @@ def check_mesh_is_fsaverage(mesh):
     with sufficient entries. Basically ensures that the mesh data is
     Freesurfer-like fsaverage data.
 
-    Used by plotting.surf_plotting.plot_img_on_surf and
-    plotting.html_surface._full_brain_info.
+    Used by plotting.surface.plot_img_on_surf and
+    plotting.surface.html_surface._full_brain_info.
```

Comment:
`nilearn.surface.surface.check_mesh_is_fsaverage` is only used by `plotting.surface.plot_img_on_surf` and `plotting.surface.html_surface._full_brain_info`, shall I move it to `plotting.surface` package?

Reply:
maybe not: we may still want to run this check even if we have not intention of plotting

Reply:
Shall I completely remove these two lines then? I'm not sure if it is a good idea to put such information in docstring.

Reply:
yes definitely misleading
better to remove them